src/index.html

```diff
@@ -17,6 +17,10 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
```

Comment:
BEM modifier class 'stopwatch--speed-up' should be used on the element that already has the base 'stopwatch' class, only if there's a need to modify the base styling for a particular state or variation.